citing-us/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
```

Comment:
Whichever URL and menu title we pick, it needs to be consistent across the board - at the moment we have 'citing OME' here and 'citing Us' everywhere else